package.json

```diff
@@ -92,7 +92,7 @@
     "eslint-plugin-vue": "^4.0.0",
     "file-loader": "^1.1.4",
     "glob-fs": "^0.1.7",
-    "jest": "^23.3.0",
+    "jest": "^23.6.0",
```

Comment:
What is the reason for updating jest? Please remove this if it is unintentional.

Reply:
This was unintentional - corrected.